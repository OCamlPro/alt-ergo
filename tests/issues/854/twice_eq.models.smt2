(set-logic ALL)
(set-info :smt-lib-version 2.6)
(set-option :produce-models true)
(declare-sort intref 0)
(declare-fun intrefqtmk (Int) intref)
(declare-fun another_mk (Int) intref)
(declare-fun a () Int)
(define-fun aqtunused () intref (intrefqtmk a))
(assert (= aqtunused (another_mk a)))
(declare-fun a1 () Int)
(assert (not (and (<= 5 a1) (<= a1 15))))
(check-sat)
(get-model)
