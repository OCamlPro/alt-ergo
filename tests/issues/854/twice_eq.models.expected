
unknown
(
  (define-fun a () Int 0)
  (define-fun intrefqtmk ((_arg_0 Int)) intref (as @a10 intref))
  (define-fun another_mk ((_arg_0 Int)) intref (as @a10 intref))
  (define-fun a1 () Int 0)
)
